Clients/GUI/Components/Wizard.razor

```diff
@@ -0,0 +1,26 @@
+﻿<CascadingValue Value="this">
+    <div id="@Id">
+        <ul class="nav nav-pills nav-justified">
+            @foreach (var step in Steps)
+            {
+                <li id="step-@(StepsIndex(step) + 1)" class="nav-item">
+                    <a class="nav-link @((ActiveStep == step) ? "active" : "")" href="javascript: void(0)"
+                       @onclick="@(e=> SetActive(step))">@step.Name</a>
```

Comment:
Hm, not sure, but do you have to write it as a lambda?

Reply:
fixed